heracal/omni.py

```diff
@@ -219,8 +219,40 @@ def compute_reds(nant, pols, *args, **kwargs):
             reds += [[(Antpol(i, pi, nant), Antpol(j, pj, nant)) for i, j in gp] for gp in _reds]
     return reds
 
-
-def aa_to_info(aa, pols=['x'], fcal=False, **kwargs):
+def minVreds(reds):
+    '''
+    Manipulate redundancy array to combine crosspols
+    into a single redundancy array - imposing that
+    Stokes V = 0.
+
+    This works in the simple way that it does because
+    of the way the reds arrays are constructed in
+    aa_to_info when 4 polarizations are present: it
+    reprsents them as 4 co-located arrays in (NS,EW) and
+    displaced in z, with the cross-combinations (e.g. 
+    polarization xy and yx) _always_ in the middle.
+    
+    Args:
+        reds: the array of redundant baseline sets.
+        
+    Return:
+        _reds: the adjusted array of redundant baseline sets.
+    '''
+    _reds = []
+    L = len(reds)
+    try:
+        assert(L%4)==0
+    except AssertionError:
+        raise Exception('Wrong number of polarizations supplied')
```

Comment:
Instead of this stanza, just write:
```
if L % 4 != 0:
   raise Exception(...)
```
Also, the exception text should be more on-point about the check that's being performed: "expected an array with a multiple of 4 elements" or something like that.